### PR TITLE
Installation

### PR DESCRIPTION
**Pull request by EmilieSchott**
Hello,
Could you check my project basis, please ? 
Thank you very much.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -17,3 +17,13 @@
...
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
+# Remove # in front of one of the following lines and complete datas :
+#
+# DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
+# DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
+# DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
+###< doctrine/doctrine-bundle ###
```
Comment (Tiriel) [inline .env]: Comments can be removed :)

File: src/Controller/FigureFormController.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class FigureFormController extends AbstractController
+{
+    /**
+     * @Route("/form/figure", name="create_figure")
```
Comment (Tiriel) [inline src/Controller/FigureFormController.php]: Naming can be improved and rationalized :)

File: .env (hunk 1 above)
Comment (EmilieSchott) [reply .env]: Ok !

File: src/Controller/FigureFormController.php (hunk 2 above)
Comment (EmilieSchott) [reply src/Controller/FigureFormController.php]: Done :)

File: bin/console [hunk 3]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env php
```
Comment (Tiriel) [inline bin/console:1]: Did you remove this on purpose?
This should stay, it's not a true comment, it's interpreted by the CLI and meant to tell the path of the PHP exec file

**Review by Tiriel — APPROVED** [review]

**Pull request merged**